docs/getting-started.rst

```diff
@@ -0,0 +1,427 @@
+
+Configure exporters to emit spans elsewhere
+-------------------------------------------
+
+The example above does emit information about all spans, but the output is a bit hard to read.
+In common cases, you would instead *export* this data to some consuming service, to be visualized and
+easily queryable. It is also common to aggregate span and trace information from multiple services into
+a single database, so that actions that require multiple services can still all be visualized together.
+
+This concept is known as distributed tracing. One such distributed tracing backend is known as Jaeger.
+
+The Jaeger project provides an all-in-one docker container that provides a UI, database, and consumer. Let's bring
+it up now:
+
+.. code-block:: sh
+
+    docker run -p 16686:16686 -p 6831:6831/udp jaegertracing/all-in-one
+
+This will start Jaeger on port 16686 locally, and expose Jaeger thrift agent on port 6831. You can visit it at http://localhost:16686.
+
+With this backend up, your application will now need to export traces to this system. ``opentelemetry-sdk`` does not provide an exporter
+for Jaeger, but you can install that as a separate package:
+
+.. code-block:: sh
+
+    pip install opentelemetry-ext-jaeger
+
+Once installed, update your code to import the Jaeger exporter, and use that instead:
+
+.. code-block:: python
+
+    # /tmp/example.py
+    from opentelemetry import trace
+    from opentelemetry.ext import jaeger
+    from opentelemetry.sdk.trace import TracerProvider
+    from opentelemetry.sdk.trace.export import SimpleExportSpanProcessor
+
+    trace.set_tracer_provider(TracerProvider())
+
+    jaeger_exporter = jaeger.JaegerSpanExporter(
+        service_name="my-helloworld-service", agent_host_name="localhost", agent_port=6831
+    )
+
+    trace.get_tracer_provider().add_span_processor(
+        SimpleExportSpanProcessor(jaeger_exporter)
+    )
+    tracer = trace.get_tracer(__name__)
+    with tracer.start_as_current_span('foo'):
+        with tracer.start_as_current_span('bar'):
+            with tracer.start_as_current_span('baz'):
+                print("Hello world from OpenTelemetry Python!")
+
+Run the script again:
+
+.. code-block:: python
+
+    python /tmp/example.py
+
+You can then visit the jaeger UI, see you service under "services", and find your traces!
+
+Integrations example with Flask
+-------------------------------
+
+The above is a great example, but it's very manual. Within the telemetry space, there are common actions that one 
+wants to instrument:
+
+* HTTP respones from web services
+* HTTP requests from clients
+* database calls
+
+To help instrument common scenarios, opentelemetry also has the concept of "instrumentations": packages that are designed to interface
+with a specific framework or library, such as Flask and psycopg2. A list of the currently curated extension packages can be found :scm_web:`here <ext/>`.
```

Comment:
Link doesn't display correctly

Reply:
That link is designed only to be rendered in read the docs, you can see how it is rendered here https://opentelemetry-python-yusuket.readthedocs.io/en/latest/getting-started.html#integrations-example-with-flask.